content/en/docs/04/_index.md

```diff
@@ -169,10 +169,10 @@ kubectl create -f <path to ingress.yaml> --namespace <namespace>
 ```
 
 {{% onlyWhenNot mobi %}}
-Afterwards, we are able to access our freshly created Ingress at `http://example-web-go-<namespace>.<domain>`
+Afterward, we are able to access our freshly created Ingress at `http://example-web-go-<namespace>.<domain>`
```

Comment:
as above

Reply:
American vs British English

Reply:
According to dict.leo.org, it's used equally in American and British English: https://dict.leo.org/german-english/afterwards

My point here is: I don't see the point of changing something that's already correct and make the text inconsistent. Now, we have both variants.